src/managers/ChannelManager.js

```diff
@@ -19,18 +19,18 @@ class ChannelManager extends CachedManager {
    * @name ChannelManager#cache
    */
 
-  _add(data, guild, cache = true, allowUnknownGuild = false) {
+  _add(data, guild, cache = true, allowUnknownGuild = false, fromInteraction = false) {
```

Comment:
At what point should we just make these options an object? @iCrawl

Reply:
Haha, probably.

Reply:
Yeah, I was so close to just doing that.